.github/workflows/pr_presubmit.yaml

```
@@ -18,6 +18,7 @@ jobs:
       with:
         ref: ${{ github.base_ref }}
         path: base_checkout
-      # `github.event.pull_request.labels.*.name` gives us the names of all labels for the PR
-    - run: go test ./ci -added="${{ steps.files.outputs.added }}" -modified="${{ steps.files.outputs.modified }}" -labels=${{ join(github.event.pull_request.labels.*.name, ' ') }} -base-dir="base_checkout" -tags=ci
-
+      # `steps.get_labels.outputs.*` gives us the names of all labels for the PR
+    - run: echo "::set-output name=labels::$(go run ci/get_labels/main.go -url=${{ github.event.pull_request.url }})"
```

You can run the whole package here.
```suggestion
    - run: echo "::set-output name=labels::$(go run ci/get_labels -url=${{ github.event.pull_request.url }})"
```